### PR TITLE
Fixed bug with heroku deployment.

### PR DESCRIPTION
**Pull request by kimslor**
# Description

Changed the start script to use node instead of nodemon.
Added dev option to use nodemon.

## Related Issue

Solves https://github.com/SE701-T5/Backend/issues/65

## Type of change

- [x] Bug fix

## How Has This Been Tested?

- [x] Automated testing

# Checklist:

- [x] Does a similar (open or closed) pull request not already exist?
- [x] Is the pull request _head_ repository a fork repository?
- [x] Is the pull request _compare_ branch a development branch?
- [x] Is the code documented, particularly in hard-to-understand areas?
- [x] Does the code build without new warnings?
- [x] Has testing been performed that proves changes are effective and work?
- [x] Has a self- and/or peer-review of the code been performed?
- [x] Have dependent changes been merged and published in downstream modules?
- [x] Does all new and existing automated testing pass?
- [x] Is the person responsible for the repository assigned to the pull request?
- [x] Is the pull request linked to a project?
- [x] Is the pull request linked to a milestone?

For more information, refer to the Contributing Guidelines and Code of Conduct links at the bottom of this page.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "main": "server.js",
   "scripts": {
     "test": "mocha ./test/* --timeout 10000 --exit",
-    "start": "nodemon server.js"
+    "start": "node server.js",
+    "dev": "nodemon server.js"
```
Comment (R055A) [inline package.json:9]:
```suggestion
```
Local development won't have any effect on the Heroku deployment

**Review by R055A — CHANGES_REQUESTED** [review]
Awesome! Thank you. There is just one suggestion to consider, please.

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "scripts": {
     "test": "mocha ./test/* --timeout 10000 --exit",
-    "start": "nodemon server.js"
+    "start": "node server.js",
```
Comment (R055A) [inline package.json:8]:
```suggestion
    "start": "node server.js"
```
Woops, forgot to remove the comma

**Review by R055A — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (christopher-alba) [reply package.json]: I think this is just for developers to be able to use Nodemon on their local repos right?

**Review by christopher-alba — APPROVED** [review]
LGTM!

**Review by R055A — APPROVED** [review]

**Pull request merged**